formatter/fmt

```diff
@@ -47,9 +47,13 @@ npx prettier \
 echo 'Done formatting with prettier'
 
 echo 'Start formatting with eslint'
-npx eslint \
-  --fix \
-  {server/app/assets/javascripts,server/tailwind.config.js,test-support,browser-test}
+cd server
+npm install --silent
+npx eslint --fix "app/assets/javascripts/**/*.ts"
+cd ../browser-test
+yarn install --silent
```

Comment:
[future improvement] I'd love to see us move to only use one-of yarn or npm

Reply:
Yeah. It's annoying to have both.